### DISCUSSION TITLE
What can kind of "model" is the one generated?

### DISCUSSION THREAD

**Discussion by agarzon** [glucauze/sd-webui-faceswaplab#93] [General]
I see it is possible to generate a "model" using a bunch of images, the training is super fast (seconds) compared with any other known method (TI, lora, dreambooth, etc) which takes hours.

I wonder, what kind of file is that **Face.safetensors** file generated, it is posible to be used in other instance like embedding or lora? or is it possible to convert it ?

Asking cause it would be awesome to have embed faces to be used in any checkpoint and style by only using one or two fotos and in seconds (photobooth project)

**Comment by glucauze** [comment]
The model is just the face embedding + gender + age data produced by buffalo_l models. It's not a model in the sense of Lora models or TI, it's more a checkpoint. See  "What is a face checkpoint?" in https://glucauze.github.io/sd-webui-faceswaplab/faq/

I average the face embeddings to get an average face.

**Reply by agarzon** [reply]
Thanks for clarifying.
